### ISSUE TITLE
Create scheduled tasks to optimize tables

### ISSUE THREAD

**Issue by elhoim** [MISP/MISP#395]
Reference on why and how: http://dev.mysql.com/doc/refman/5.1/en/optimize-table.html

**Comment by iglocska** [comment]
Good idea. A general cleanup task would be needed for some other areas too!

**Comment by elhoim** [comment]
This exists in current 2.5: `cake Admin optimiseTables` runs `OPTIMIZE TABLE` over every table with a progress bar — `app/Console/Command/AdminShell.php:1225`, registered as a subcommand at `app/Console/Command/AdminShell.php:132`. Scheduling is delegated to the system cron rather than the legacy `scheduled_tasks` table, which is the current MISP pattern for maintenance jobs.

There is also a targeted variant for the access log table at `app/Model/AccessLog.php:162`.

Recommend closing.

**Comment by elhoim** [comment]
Implemented. `AdminShell::optimiseTables()` (`app/Console/Command/AdminShell.php:1225`) is exposed as a console subcommand, and `AccessLog` runs `OPTIMIZE TABLE` on rotation (`app/Model/AccessLog.php:162`). Closing as done.
